### ISSUE TITLE
Authentication Server: Replace text with Yanovis Icon

### ISSUE THREAD

**Issue by mmalfertheiner** [noi-techpark/authentication-server#68]
https://applogin.com/img/logo_applogin_white.svg

**Comment by mmalfertheiner** [comment]
@Piiit Yanovis requested to change the login button to this icon. Is this ok?

**Comment by Piiit** [comment]
@mmalfertheiner I think Stefano wanted to switch all login buttons to a text-only representation...

@sseppi What do you say to this?

**Comment by sseppi** [comment]
@Piiit yes at the moment I would suggest to go to a text-only representation.
